src/i18n/NL.mjs

```diff
@@ -1,9 +1,56 @@
 import Num2Word_Base from '../classes/Num2Word.mjs';
 
-export default function() {
+export default function(options) {
   Num2Word_Base.call(this);
 
-  this.cards = [{'1000000000000000000000000000': 'quadriljard'}, {'1000000000000000000000000': 'quadriljoen'}, {'1000000000000000000000': 'triljard'}, {1000000000000000000: 'triljoen'}, {1000000000000000: 'biljard'}, {1000000000000: 'biljoen'}, {1000000000: 'miljard'}, {1000000: 'miljoen'}, {1000: 'duizend'}, {100: 'honderd'}, {90: 'negentig'}, {80: 'tachtig'}, {70: 'zeventig'}, {60: 'zestig'}, {50: 'vijftig'}, {40: 'veertig'}, {30: 'dertig'}, {20: 'twintig'}, {19: 'negentien'}, {18: 'achtien'}, {17: 'zeventien'}, {16: 'zestien'}, {15: 'vijftien'}, {14: 'veertien'}, {13: 'dertien'}, {12: 'twaalf'}, {11: 'elf'}, {10: 'tien'}, {9: 'negen'}, {8: 'acht'}, {7: 'zeven'}, {6: 'zes'}, {5: 'vijf'}, {4: 'vier'}, {3: 'drie'}, {2: 'twee'}, {1: 'een'}, {0: 'nul'}];
+  const defaultOptions = {
+    includeOptionalAnd: false,
+    noHundredPairs: false,
+  };
+
+  const localOptions = Object.assign({}, defaultOptions, options);
```

Comment:
No need for an empty object in the first param.

To shorten everything up I usually do something like this.

```javascript
export default function(options) {
  options = Object.assign({
    includeOptionalAnd: false,
    ...
  }, options);
}
```